schedule/yast/minimal+base/minimal+base@yast.yaml

```diff
@@ -24,7 +24,7 @@ schedule:
   - installation/authentication/default_user_simple_pwd
   - installation/resolve_dependency_issues
   - installation/select_patterns
-  - installation/installation_overview
+  - installation/installation_settings/validate_default_target
```

Comment:
we don't do anything here: https://openqa.suse.de/tests/7924011#step/installation_overview/1.

Reply:
Removed.